docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksNsxtDataSourceCredentials.md

```diff
@@ -0,0 +1,179 @@
+# Update-AriaNetworksNsxtDataSourceCredentials
+
+## Synopsis
+
+Update credentials for a NSX Manager data source in VMware Aria Operations for Networks.
+
+## Syntax
+
+### credentials (Default)
+
+```powershell
+Update-AriaNetworksNsxtDataSourceCredentials -fqdn <String> -id <String> -collectorId <String>
+ -nickname <String> [-username <String>] [-password <String>] [<CommonParameters>]
+```
+
+### certificate
+
+```powershell
+Update-AriaNetworksNsxtDataSourceCredentials -fqdn <String> -id <String> -collectorId <String>
+ -nickname <String> [-certificate <String>] [-privatekey <String>] [<CommonParameters>]
+```
+
+## Description
+
+The `Update-AriaNetworksNsxtDataSourceCredentials` cmdlet allows a user to update credentials for a NSX Manager data source in VMware Aria Operations for Networks.
+
+## Examples
+
+### Example 1
+
+```powershell
+Update-AriaNetworksNsxtDataSourceCredentials -fqdn sfo-m01-nsx01.sfo.rainpole.io -id 15832:904:7312957441829059413 -CollectorId 15832:901:1711011916294613031 -nickname "sfo-m01-vc01 - Management Domain vCenter Server" -certificate F:\certs\sfo-m01-nsx01.cer -privatekey F:\certs\sfo-m01-nsx01.key
+This example updates the credentials for the NSX Manager data source in VMware Aria Operations for Networks to use a new certificate and private key for a NSX principal identity user.
+```
+
+### Example 2
+
+```powershell
+Update-AriaNetworksNsxtDataSourceCredentials -fqdn sfo-m01-nsx01.sfo.rainpole.io -id 15832:904:7312957441829059413 -CollectorId 15832:901:1711011916294613031 -nickname "sfo-m01-vc01 - Management Domain vCenter Server" -username svc-inv-vsphere -password VMw@re1!VMw@re1!
+This example updates the credentials for the NSX Manager data source in VMware Aria Operations for Networks to use a new NSX service account user with a password.

```

Comment:
Move the description of the example outside of the code block. 🙃

Reply:
Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/cd3f7c173de4e0535b7ea866786ba1f78d1ae24e